docs/logagent/plugins.md

```diff
@@ -1,4 +1,4 @@
-title: Logagent plugins 
+!title: Logagent plugins 
```

Comment:
what is this `!` ?

Reply:
A typo. See https://squidfunk.github.io/mkdocs-material/extensions/metadata/